app/infrastructure/exceptions/error_type.py

```
@@ -0,0 +1,66 @@
+    COURSE_NOT_EXIST = (
+        "강좌 미존재",
+        "강좌가 존재하지 않아요.",
+        status.HTTP_404_NOT_FOUND,
+    )
+    TIMETABLE_NOT_EXIST = (
+        "시간표 미존재",
+        "시간표가 존재하지 않아요.",
+        status.HTTP_404_NOT_FOUND,
+    )
+
+    def __init__(self, title: str, detail: str, status_code: status):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)